pkg/transportrequest/gitutils.go

```diff
@@ -2,12 +2,52 @@ package transportrequest
 
 import (
 	"fmt"
+	pipergit "github.com/SAP/jenkins-library/pkg/git"
+	"github.com/SAP/jenkins-library/pkg/log"
 	"github.com/SAP/jenkins-library/pkg/piperutils"
+	"github.com/go-git/go-git/v5"
 	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/pkg/errors"
+	"os"
 	"regexp"
 	"sort"
 )
 
+// needs to be replaced by mocks in the tests
+var getWorkDirectory = os.Getwd
+
+// FindIDInRange finds a ID according to the label in a commit range <from>..to.
```

Comment:
`FindIDInRange` errors in case there is no ID or more than one ID. This is some common coding which we will need creating/releasing a transportRequest in the SOLMAN use-case. Purpose is to avoid having the same coding several times ...

I'm open for other suggestions ...